### PR TITLE
fix: small change in instructions for showcase app

### PR DESCRIPTION
**Pull request by psturc**
ping @jstaffor 

Here's a small fix and the suggestion on we can instruct a user to deal with the `mobile-services.json` file. Configuration file content is not repeated in docs then.

Feel free to modify it/rephrase it

### CODE REVIEW

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 1]
```diff
@@ -109,9 +109,8 @@ xref:setting-up-mobile-services-to-demonstrate-showcase-apps[Setting up Mobile S
 		}
 	}]
 }
-
-module.exports = config;
 ----
+. Copy the content of this configuration to your clipboard
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]:
```suggestion
. Copy the content to your clipboard.
```

File: modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc [hunk 2]
```diff
@@ -128,29 +127,7 @@ xref:creating-the-data-sync-mobile-app-binding[Creating the Data Sync Mobile App
...
-    "config": {
-      "websocketUrl": "wss://sync-app-syncshowcasetestbed.com/graphql"
-      }
-}
-  ]
-};
-
-module.exports = config;
-----
+2. Paste the Data Sync Mobile App configuration file data (previously copied to clipboard) into the *mobile-services.json* file located within _src_
```
Comment (jstaffor) [inline modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc]:
```suggestion
2. Paste the Data Sync Mobile App configuration content into the *mobile-services.json* file located within _src_
```

**Comment by jstaffor** [comment]
Hi @psturc - I've reviewed this PR :+1:

**Comment by psturc** [comment]
@jstaffor feel free to merge if you're happy with the changes.

**Comment by jstaffor** [comment]
@jstaffor - this feeds into https://issues.jboss.org/browse/AEROGEAR-9364

**Pull request merged**